### PR TITLE
Add maintainer script

### PR DESCRIPTION
**Pull request by huayunh**
For "As Erice, I'd like to search icons by their aliases on component sticker sheet"

#### Changes proposed in this Pull Request:
- add maintainer's script and readmes
- I really have no idea how to do a proper import / require in the main `code.ts`, so I ended up hacking it in a dirty way. It ain't elegant but got the job done. 🤷‍♀️
- Changes have been pushed to Figma and published for both Material and PXBlue icons.

#### To Test:
- Follow the instructions at `_maintenance_/icon-description/README.md`

### CODE REVIEW

File: _maintenance_/icon-description/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Icon Description
+
+By default, this maintenance plugin will update the "description" field for each selected icon using the array of tags supplied in meta files. It will also output some useful information in Figma's developer console.
+
+## For Maintainers
+
+The developer functionality only works through Figma desktop app.
+
+To sync up to Material Design and PX Blue's latest icon meta file, open Component Stickersheet's icon page (I suggest doing this from a [Figma branch](https://help.figma.com/hc/en-us/articles/360063144053-Create-branches-and-merge-changes)), and link to this plugin's `manifest.json`. Observe that the plugin shows up under "Plugin > Development > Icon Description" in Figma/
```
Comment (joebochill) [inline _maintenance_/icon-description/README.md:9]:
```suggestion
To sync up to Material Design and PX Blue's latest icon meta file, open Component Stickersheet's icon page (I suggest doing this from a [Figma branch](https://help.figma.com/hc/en-us/articles/360063144053-Create-branches-and-merge-changes)), and link to this plugin's `manifest.json`. Observe that the plugin shows up under "Plugin > Development > Icon Description" in Figma.
```

File: _maintenance_/icon-description/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+**To update Material Design icons:** You will need to do `yarn && yarn build`, select those Material icon components you would like to update, and run the plugin.
```
Comment (joebochill) [inline _maintenance_/icon-description/README.md:11]: Should we say something in here about updating the meta file? Won't that need to be updated or else the plugin just uses the existing hardcoded file?

**Review by joebochill — COMMENTED** [review]
do the built js files need to be checked in or can we add them to the gitignore?

**Comment by huayunh** [comment]
Why is my git ignore not working...

**Comment by joebochill** [comment]
> Why is my git ignore not working...

since you have already committed the files, you probably have to do `git rm` to explicitly remove them (gitignore just ignores _changes_ to the file). Or you might need to change `**/build` to `**/**/build` — I can't recall if '**' does any nested folder or if it only looks one level deep.

**Comment by huayunh** [comment]
Problems all cleared

**Comment by huayunh** [comment]
one moment, I just had an idea

**Comment by huayunh** [comment]
now i may rest...

**Review by joebochill — APPROVED** [review]

**Pull request merged**